src/mui/input/SelectManyInput.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component, PropTypes } from 'react';
+import ChipInput from 'material-ui-chip-input';
+import FieldTitle from '../../util/FieldTitle';
+
+/**
+ * An Input component for a array
+ *
+ * @example
+ * <SelectManyInput source="first_name" />
```

Comment:
bad example. How can `first_name` be an array? Same for the rest of the comments for that component.